src/components/Comments.jsx

```diff
@@ -14,6 +15,30 @@ export default function Comments({id, comments, setComments}) {
     })
     }, [])
 
+    function deleteCommentAPI(id) {
+        api.deleteCommentById(id)
```

Comment:
You should add a loading state here to control whether a user can press the delete button multiple times while a request is still being made.